README.md

```diff
@@ -1,8 +1,27 @@
-Confluent's Apache Kafka client for Golang
-==========================================
+Confluent's Golang Client for Apache Kafka<sup>TM</sup>
```

Comment:
I anguished over this change for a good 30 minutes. The problem is i think the line below is very important (that it's a client for Kafka and Confluent Platform), and it's very difficult to write that if we put it the way around that you suggest (which I too think makes more sense). Worth noting that a lot of other people say it the way I changed it to (eg https://github.com/dpkp/kafka-python - python client for Apache Kafka).